config/initializers/carrierwave.rb

```diff
@@ -11,9 +11,7 @@ def local_storage_config
     config.storage = :file
     config.enable_processing = !Rails.env.test? # disabled for test
     config.asset_host = if Rails.env.production?
-                          "https://#{ApplicationConfig['APP_DOMAIN']}/localimages"
-                        elsif Images::Optimizer.imgproxy_enabled?
-                          "http://#{ApplicationConfig['APP_DOMAIN']}"
+                          "https://#{ApplicationConfig['APP_DOMAIN']}"
```

Comment:
Do we have any production instances already using the /localimages/ ?

It looks like local storage only happens if we're not using S3 (presumably the two heroku instances), not in a FOREM_CONTEXT (like cloud fleet are) so I don't think this impacts anything for anyone except self-hosted. If I misunderstand the impact we might want to be more careful.

Reply:
confirmed with @jdoss that all forem cloud deployments are unaffected (they have S3 storage and FOREM_CONTEXT set)